README.md

```diff
@@ -61,14 +62,23 @@ Update any relevant keys in your new .env file. For example, if you will be work
 
 Please note: Any time you change the values in this .env file, you will need to stop and restart your Docker container in order for the new values to take effect.
 
-#### Install Docker Compose on Mac
+#### Install Docker Desktop
+
+- If using Windows, before installing Docker Desktop make sure you have installed WSL (Windows Subsystem for Linux). Else, install WSL first: https://docs.microsoft.com/en-us/windows/wsl/install
+- Click https://docs.docker.com/desktop/, to start to install Docker Desktop.
+- In the left sidebar, choose between Mac or Windows installation and follow the instructions.
```

Comment:
There are now installations for *nix, win & mac likewise. Maybe this information can be explored and added here.